src/publish/confluence/confluence-helper.ts

```diff
@@ -597,7 +603,9 @@ export const updateLinks = (
 
         updated = updated.replace(linkFullFileName, pagePath);
       } else {
-        console.warn(`Link not found for ${siteFilePath}`);
+        if (!collectedPass2Changes.includes(changeToProcess)) {
```

Comment:
If you used `collectedPass2Changes.push(changeToProcess);` here, you could get away with making `collectedPass2Changes` a const above.

Reply:
Thanks for the suggestion, stylistically I prefer clean mutation of lets rather than "hidden" mutation of consts.  I need to think more about this though to be honest, I don't think it is quite there.

Reply:
After a second look, I think I'm going to keep it as this is updating the let in the closure - pretty non-standard.  Thank you again for the idea.